test.py

```diff
@@ -160,8 +160,16 @@ def spreadTest(generator, expected):
         # sort for easy comparison
         actual[j] = ''.join(sorted(actual[j]))
 
-    assert arraysEqual(expected, actual), "In a large sample of generated slugids, the range of characters found per character position in the sample did not match expected results.\n\nExpected: " + str(expected) + "\n\nActual: " + str(actual)
+    arraysEqual(expected, actual), "In a large sample of generated slugids, the range of characters found per character position in the sample did not match expected results.\n\nExpected: " + str(expected) + "\n\nActual: " + str(actual)
 
 def arraysEqual(a, b):
     """ returns True if arrays a and b are equal"""
-    return cmp(a, b) == 0
+    assert len(a) == len(b)
+    fails = False
+    msg = []
+    for x in range(0, len(a)):
+      if 0!= cmp(a[x], b[x]):
```

Comment:
* nit: can you add a space (` `) after the `0`?
* nit: I'd prefer `if cmp(a[x], b[x]) != 0` to `if 0 != cmp(a[x], b[x]):`